pkg/server/deploy/deploy.go

```diff
@@ -38,7 +38,7 @@ type Operations interface {
 type K8sOperations interface {
 	CreateOrUpdateDeploy(deploySpec *spec.Deploy) error
 	CreateOrUpdateCronJob(cronJobSpec *spec.CronJob) error
-	ExposeDeploy(namespace, name, svcType, portName string, vHosts []string, reserveStaticIp bool, w io.Writer) error
+	ExposeDeploy(namespace, name, svcType, portName string, vHosts []string, reserveStaticIp bool, ingressClass string, w io.Writer) error
```

Comment:
What about creating a new struct to using as input parameters of this method?

Reply:
Good idea.

Reply:
Oops, looking at it, I would have to change it in a lot of places. I prefer to open another PR for that.